libsnowflakeclient/CMakeLists.txt

```diff
@@ -21,7 +21,10 @@ set(SOURCE_FILES
         lib/connection.c
         lib/results.c
         lib/uuid4.c
-        lib/basic_types.c)
+        lib/basic_types.c
+        lib/error.h
+        lib/error.c
+        lib/snowflake_client_int.h)
```

Comment:
Same here

Reply:
The error header file I can remove. For snowflake_client_int.h, the entire definition exists in the header file (and there is no source file) so I need to include that as a "source file" to get this to work.

Reply:
hmm, ok.